Tribler/community/tunnel/crypto/tunnelcrypto.py

```diff
@@ -43,13 +43,17 @@ def verify_and_generate_shared_secret(self, dh_secret, dh_received, auth, B):
         return shared_secret
 
     def generate_session_keys(self, shared_secret):
+        # TODO hkdf???
```

Comment:
@Pathemeous I read the wiki yesterday but didn't comment yet, thanks for the link!